lib/assets/ContentSecurityPolicy.js

```diff
@@ -0,0 +1,116 @@
+var util = require('util'),
+    _ = require('lodash'),
+    matchSourceExpression = require('../matchSourceExpression'),
+    extendWithGettersAndSetters = require('../util/extendWithGettersAndSetters'),
+    Text = require('./Text');
+
+function toCamelCase(str) {
```

Comment:
Ah, I didn't realize that existed. Then again, it took half a minute to type the function, and I'm fairly certain it's bug-free and won't ever have to be updated. Pulling in a dependency feels like overkill.